vsphere/resource_vsphere_resource_pool_test.go

```diff
@@ -0,0 +1,222 @@
+package vsphere
```

Comment:
There are some missing tests:

* A test for checking that a resource pool can be moved between different roots (with checks that validate the correct parent)
* An import test
* Tests for tagging (see comment in CRUD file regarding tags)